src/main/scala/org/downtowndailybread/bethsaida/model/ScheduleDetail.scala

```diff
@@ -1,6 +1,59 @@
 package org.downtowndailybread.bethsaida.model
 
+import java.time.{LocalDateTime, LocalTime, ZonedDateTime}
+import java.util.TimeZone
+
+import org.dmfs.rfc5545.DateTime
+import org.dmfs.rfc5545.recur.RecurrenceRule
+
 case class ScheduleDetail(
                            rrule: String,
+                           beginTime: LocalTime,
+                           endTime: LocalTime,
+                           scheduleCapacity: Option[Int],
                            enabled: Boolean
-                         )
+                         ) {
+
+  private val rule = new RecurrenceRule(if(rrule.startsWith("RRULE:")) rrule.substring("RRULE:".size) else rrule)
```

Comment:
Why would it not start with `RRULE`?

Reply:
The spec seems to indicate that it should start with RRULE, but the library itself doesn't like that.